src/ListBox/README.md

```diff
@@ -0,0 +1,132 @@
+# ListBox
+
+Provides a ListBox component that takes ListBoxOption components as children and handles the selection of and navigation through the ListBoxOption components.
+
+## API
+
+### `children` (ListBoxOption component array; required)
+`children` specifies the list of ListBoxOption components that will be displayed within the ListBox element.
+
+### `className` (string; optional)
+`className` specifies Bootstrap class names to apply to the ListBox component. The default is an empty string.
+
+### `selectedOptionIndex` (number; optional)
```

Comment:
Actually, a question I have is how a user could clear the selected ListBoxOption using this prop. Would they pass in a negative integer? A null? I'd like to support that case, since there have been multiple instances of us needing a way to clear a components state this way.